README.md

```diff
@@ -60,7 +68,7 @@ From the root of your repository directory, run `./scripts/zip_lambda_code.sh`.
 You'll first need to set up the [Terraform remote state](https://www.terraform.io/docs/state/remote.html) to be the source of truth for the state of your deployed infrastructure.
 
 1. Run `python3 ./scripts/setup.py state` (this will create  an S3 bucket and DyanmoDb lock table for Terraform)
-1. Initialize and apply the infrastructure:
+2. Initialize and apply the infrastructure:
```

Comment:
@gohelankit  FYI - I like to keep this all numbered with `1.` because Markdown will parse it into an actual numbered list for you, and it makes adding / reordering the lines super easy